geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/GeodeRedisService.java

```diff
@@ -61,9 +60,6 @@ private void registerDataSerializables() {
     InternalDataSerializer.getDSFIDSerializer().registerDSFID(
         DataSerializableFixedID.REDIS_KEY,
         RedisKey.class);
-    InternalDataSerializer.getDSFIDSerializer().registerDSFID(
-        DataSerializableFixedID.REDIS_BYTE_ARRAY_WRAPPER,
```

Comment:
I think we should remove this constant from `DataSerializableFixedID` as well?

Reply:
Good catch, thanks!